packages/syft/tests/conftest.py

```diff
@@ -28,6 +29,7 @@ def faker():
 
 @pytest.fixture(autouse=True)
 def worker(faker):
+    flags.CAN_REGISTER = False
```

Comment:
By default it is false,maybe we could remove it from the worker fixture here?

Reply:
Fixed by: https://github.com/OpenMined/PySyft/pull/7715/commits/b76fd05f983d0368a46dc630c29e759307ed42e2